src/main/java/net/sf/mzmine/modules/tools/msmsspectramerge/MsMsSpectraMergeParameters.java

```diff
@@ -0,0 +1,36 @@
+package net.sf.mzmine.modules.tools.msmsspectramerge;
+
+import net.sf.mzmine.parameters.Parameter;
+import net.sf.mzmine.parameters.impl.SimpleParameterSet;
+import net.sf.mzmine.parameters.parametertypes.*;
+
+import java.text.NumberFormat;
+import java.util.Locale;
+
+public class MsMsSpectraMergeParameters extends SimpleParameterSet {
+
+    public MsMsSpectraMergeParameters() {
+        super(
+                new Parameter[]{MERGE_MODE, MZ_MERGE_MODE, INTENSITY_MERGE_MODE, MASS_ACCURACY, COSINE_PARAMETER, PEAK_COUNT_PARAMETER, ISOLATION_WINDOW_OFFSET, ISOLATION_WINDOW_WIDTH}
+        );
+    }
+
+    public static final PercentParameter COSINE_PARAMETER = new PercentParameter("Cosine threshold (%)", "Threshold for the cosine similarity between two spectra for merging. Set to 0 if the spectra may have different collision energy!", 0.7d, 0d, 1d);
+
+    public static final PercentParameter PEAK_COUNT_PARAMETER = new PercentParameter("Peak count threshold ", "After merging, remove all peaks which occur in less than X % of the merged spectra.", 0.2d, 0d, 1d);
+
+    public static final IntegerParameter MASS_ACCURACY = new IntegerParameter("expected mass deviation (ppm)", "Expected mass deviation of your measurement in ppm (parts per million). We recommend to use a rather large value, e.g. 10 for Orbitrap, 15 for Q-ToF, 100 for QQQ.", 15, 1, 100);
```

Comment:
This should be changed to MZToleranceParameter, which is used throughout MZmine (allows both relative and absolute tolerances)